app/components/settings-form.js

```diff
@@ -0,0 +1,27 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  image: '',
+  username: '',
+  bio: '',
+  email: '',
+  password: '',
+
+  actions: {
+    submit() {
+      const { image, username, bio, email, password } = this;
```

Comment:
Hmm, should we be using `this.getProperties` instead of just `this` here?

Reply:
I would switch to using `getProperties` if there were side effects to this approach.